src/Sfa.Tl.Matching.Models/Dto/SendProviderFeedbackEmail.cs

```diff
@@ -1,7 +1,26 @@
-﻿namespace Sfa.Tl.Matching.Models.Dto
+﻿using System.Collections.Generic;
+
+namespace Sfa.Tl.Matching.Models.Dto
 {
     public class SendProviderFeedbackEmail
     {
         public int BackgroundProcessHistoryId { get; set; }
     }
+
+    public class SendProviderReferralEmail
```

Comment:
move this class to its own separate file

Reply:
done